brain/detail/targets.h

```diff
@@ -0,0 +1,79 @@
+/* Copyright (c) 2013-2018, EPFL/Blue Brain Project
+ *                          Juan Hernando <jhernando@fi.upm.es>
+ *
+ * This file is part of Brion <https://github.com/BlueBrain/Brion>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+#pragma once
+
+#include <brion/blueConfig.h>
+#include <brion/target.h>
+
+#include <lunchbox/log.h>
+
+namespace brain
+{
+class Targets
```

Comment:
It's sad that is still needed/used, given that the target system is somewhat deprecated for years. I would not enable people by providing support for it. A single target file is more than enough, and the user should provide a custom target file to fit its needs.

Reply:
As discussed in person, a single target file may not be sufficient in all cases. I'll keep this change even if only for the refactoring. The multi-target thing may can back later.